### ISSUE TITLE
Annotations do not change from failed to passed when a commit is pushed

### ISSUE THREAD

**Issue by joehoppe** [ArtiomTr/jest-coverage-report-action#389]
### Describe a bug

If I run my pipeline and the tests fail, the annotations work great. If my integration tests fail, and I push another commit that fixes them, the annotations build step (if I'm using the right terminology) does not get updated to say that they passed, even though the job passed. I verified in the logs that the job is successful. The build gate then reports "Merging is blocked' because the annotations step is still marked as failed

For this use case, please note that I'm running your plugin twice under a single job. 

      - name: 🧪 Run unit tests 
        uses: ArtiomTr/jest-coverage-report-action@v2
        with:
          annotations: failed-tests
          test-script: npm run test-coverage
          custom-title: Unit Test Coverage

      - name: 🧪 Run integration tests 
        uses: ArtiomTr/jest-coverage-report-action@v2
        with:
          annotations: failed-tests
          test-script: npm run int-test-coverage
          custom-title: Integration Test Coverage

Thanks!!!

### Expected behavior

After pushing a commit that makes all integration tests pass, the build gate should be marked as succeeded.

### Details

-   Action version:
-   OS, where your action is running (windows, linux): ubuntu
-   <details><summary>action.yml file</summary>

        ```yml
        on:
  workflow_call:

    inputs:
      environment:
        required: true
        type:     string
     
jobs: 
  unit-tests:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout release branch
        uses: actions/checkout@v3

      - name: 📗 Use node.js 18.x
        uses: actions/setup-node@v3
        with:
          node-version: 18.x

      - name: 🧪 Run unit tests 
        uses: ArtiomTr/jest-coverage-report-action@v2
        with:
          annotations: failed-tests
          test-script: npm run test-coverage
          custom-title: Unit Test Coverage

        ...

      - name: 🧪 Run integration tests 
        uses: ArtiomTr/jest-coverage-report-action@v2
        with:
          annotations: failed-tests
          test-script: npm run int-test-coverage
          custom-title: Integration Test Coverage

          ...
        ```

    </details>

-   <details><summary>Screenshots</summary>

        

        

![image](https://github.com/ArtiomTr/jest-coverage-report-action/assets/4017029/efd66100-31d7-4ffd-bbc5-fcae5b1c9334)

    </details>

**Comment by hyperair** [comment]
I think the issue is that `jest-coverage-report-action` doesn't publish a checkrun if there are no annotations, and annotations only exist when there are test failures.

**Comment by ArtiomTr** [comment]
Fix published in `v2.2.9`
